src/components/ColorChoiceDialog.js

```diff
@@ -32,7 +32,22 @@ function ColorChoiceDialog(props) {
   }
 
   function handleSubmit() {
-    onClose({ red: red, green: green, purple: purple });
+    if (red === green || green === purple || red === purple) {
+      alert("All colors must be unique");
+    } else onClose({ red: red, green: green, purple: purple });
```

Comment:
I've seen a few people set identical colors on purpose for a laugh, and haven't heard of anyone doing it accidentally. Seems easy to go back and fix it if done accidentally.

Have you seen demand for this feature?

Reply:
Nope lol I just personally felt like it shouldn't be allowed, but def don't think it's necessary.

Reply:
I agree that we might not want to enforce this.